_pages/ato/ownership.md

```diff
@@ -1,27 +0,0 @@
----
-title: System Ownership
-navtitle: System Ownership
-parent: ATOs
----
-
-Whoever **owns** a government information system must accept the risk of operating that system. Most 18F engagements, regardless of business unit, require us to operate an information system for some period of time: whether in alpha, beta, or fully live status. 
-
-For Federal teams that support the mission of _other_ Federal agencies, it can sometimes be unclear who must accept this risk.
-
-At 18F, we require that all of our partners agree that for 18F owned systems, 18F will enforce a risk management framework and will grant the system's **authority to operate** (ATO) based on 18F standards. If the system is transferred to the partner at later date, they will then need to issue a new ATO. This agreement provides the necessary clarity on roles and responsibilities for the system.
```

Comment:
Made an issue to revisit this later: https://github.com/18F/before-you-ship/issues/274